qcengine/programs/tests/standard_suite_ref.py

```diff
@@ -768,12 +1001,15 @@
         },
         "data": {
             "HF TOTAL ENERGY": _scf_nh2_adz_pk_rohf,
+            "HF TOTAL GRADIENT": _grad_scf_nh2_adz_pk_rohf,
             "MP2 CORRELATION ENERGY": -0.15702660833165538,
             "MP2 SINGLES ENERGY": -0.0028059971624814647,
             "MP2 SAME-SPIN CORRELATION ENERGY": -0.03466304269235235,
             "MP2 TOTAL GRADIENT": np.array(
                 [0.0, 0.0, 0.026398091851, 0.0, 0.014012163884, -0.013199045925, 0.0, -0.014012163884, -0.013199045925,]
             ).reshape((-1, 3)),
+            "MP3 CORRELATION ENERGY": -0.173406218947,  # cfour
```

Comment:
We don't currently have CI in place for cfour. Are the values derived from cfour going to be exercised at all by our current crop of tests?

Reply:
current crop, no. I'll expand the stdsuite testing here sometime to at least psi4 and cfour. but it isn't dead code, as it'll be used by https://github.com/psi4/psi4/pull/1971

Reply:
Ah cool, thank you! This sounds good to me; I don't want it to block merge of this substantial work!